presto-docs/src/main/sphinx/admin/properties.rst

```diff
@@ -515,3 +741,137 @@ The following properties allow tuning the :doc:`/functions/regexp`.
     to hit the limit on matches for subsequent rows as well, you want to use the
     correct algorithm from the beginning so as not to waste time and resources.
     The more rows you are processing, the larger this value should be.
+
+Session properties
+------------------
+
+``processing_optimization``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+ * **Type:** ``String`` (``disabled``, ``columnar`` or ``columnar_dictionary``)
+ * **Default value:** ``optimizer.processing-optimization`` (``false``)
```

Comment:
For all of these session properties that have config parameters as a default, let's write them like this:
```
* **Default value:** config ``optimizer.processing-optimization``
```
This makes it clear that it comes from a config property (especially important if we move session properties to a separate page). It also removes the duplicate default value, which is documented with the config property. I'm concerned that people will forget about this extra place when the config default is updated.